src/Web/Catalog.WebForms/readme.md

```diff
@@ -0,0 +1,80 @@
+# Running the Catalog Editor WebForms application
+
+You can run the catalog editor Web Forms application locally, using
+hard coded catalog data, or in a set of Windows based Docker Containers,
+using the catalog microservice.
+
+To run locally, you edit the [web.config](web.config) file to set the
+`usefake` flag to true:
+
+```xml
+  <appSettings>
+    <add key="usefake" value="true" />
+    <add key ="CatalogURL" value="http://catalog.api:5101" />
+  </appSettings>
+```
+
+Set that value to `false` to run with the full catalog microservice.
+
+Then, set the startup project in Visual Studio to the *Catalog.WebForms* project. Build and run, and test the application.
+
+## Configure Windows Containers
+
+The Catalog.WebForms project uses the full framework, so will only run in Windows based Docker containers. Before running in Docker, make sure you
+are running Docker with Windows containers configured. Right-click on the Docker node, and if "switch to Windows Containers" is displayed, click that. If "switch to Linux containers" is displayed, you are already running
+with Windows containers.
+
+## Update Configured IP address
+
+There is a current limitation in Docker for Windows when running with
+Windows containers where *localhost* doesn't resolve to the Docker host IP.
+Instead, you need to find the physical machine's IP address and use that for
+the external address of the services.
+
```

Comment:
This limitation is centered around communication between a container and _its host_--we don't have the routing in place for container to be accessed (via its exposed port on the host) using localhost, but we recently added a fix so that this is possible using the container host's IP address. 
I would avoid saying this is a Docker for Windows limitation, because it's really just a Windows (HNS) limitation--but you probably don't need to say that either. If you'd like to see how I generally word this, take a look at this [StackOverflow post that I replied to](http://stackoverflow.com/questions/43769806/docker-for-windows-cannot-access-service-on-exposed-port-in-windows-container-m/43770786?noredirect=1#comment74590252_43770786). I've included descriptions of the "workarounds" for this in my answer to that post.